CMakeLists.txt

```diff
@@ -142,14 +142,14 @@ endif()
 
 target_include_directories(tinycbor
   PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>"
-  PUBLIC "$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}>"
+  PUBLIC "$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}>"
   PUBLIC "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
 
 install(FILES
-  src/cbor.h
-  ${CMAKE_BINARY_DIR}/tinycbor-version.h
-  ${CMAKE_BINARY_DIR}/tinycbor-export.h
+  ${PROJECT_SOURCE_DIR}/src/cbor.h
```

Comment:
I'm surprised this one is needed. Shouldn't paths be relative to the current dir anyway?

Reply:
https://cmake.org/cmake/help/latest/command/install.html#files

> File names given as relative paths are interpreted with respect to the current source directory.

Reply:
We'll leave. I just wanted to know if I was missing something.

Thanks.